upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template

```
@@ -3854,6 +3854,10 @@ spec:
                 configMapKeyRef:
                   name: calico-config
                   key: veth_mtu
+            # Set wireguard for felixconfiguration.
+            # This environment variable is loaded in https://github.com/projectcalico/felix/blob/bbd5c2b04230b1f23006ad44fab1ed6ef79e1270/config/env_var_loader.go#L26
+            - name: FELIX_WIREGUARDENABLED
+              value: "{{ .Networking.Calico.WireguardEnabled }}"
```

Please update and move to the `# kops additions` section at the end.
https://github.com/kubernetes/kops/blob/174ba0e32338c828f1706e6845cf7da1c14d8707/upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template#L3877
```suggestion
            # Enable WireGuard encryption for all on-the-wire pod-to-pod traffic
            - name: FELIX_WIREGUARDENABLED
              value: "{{ .Networking.Calico.WireguardEnabled }}"
```

---

I resolved in https://github.com/kubernetes/kops/pull/10037/commits/b13027565ccfa3acdc2966fd51368ae1e9dd1165 .